.gitignore

```diff
@@ -13,3 +13,4 @@ docs/
 
 /app/assets/javascripts/megatron/esvg.js
 error_pages.zip
+.idea
```

Comment:
You should probably throw this into your system-wide `.gitignore`, otherwise you'll have to add it to all 10,000 of our various repos.